spec/bolt/cli_spec.rb

```diff
@@ -161,6 +161,24 @@ def stub_config(file_content = {})
             }.to raise_error(Bolt::CLIExit)
           }.to output(/Available actions are:.*upload/m).to_stdout
         end
+
+        it 'accepts puppetfile' do
+          cli = Bolt::CLI.new(%w[help puppetfile])
+          expect {
+            expect {
+              cli.parse
+            }.to raise_error(Bolt::CLIExit)
+          }.to output(/Available actions are:.*install.*show-modules/m).to_stdout
```

Comment:
This should also have a check that some options aren't included in the output. `--private-key` was the option that raised the ticket, so that might be a good one to make sure isn't in the output.

```
}.to output(/Available actions are:.*install.*show-modules/m).to_stdout
.and_not_to include(/--private-key/)
```